.github/workflows/unit-tests.yml

```diff
@@ -34,22 +34,16 @@ jobs:
       # - coverage: Whether to run the tests with code coverage.
       # - experimental: Whether the build is "allowed to fail".
       matrix:
-        php: ['7.2', '7.3', '7.4']
+        php: ['7.2', '7.3', '7.4', '8.0', '8.2', '8.3']
```

Comment:
Any reason why we have php versions different here and in above file?

Reply:
Yes! 😃

In `integration-tests.yml`, PHP versions `8.1`, `8.2` and `8.3` are defined in the `include` section in order to run the tests with specific configurations (for example code coverage or to run the tests against WP `trunk`).

In `unit-tests.yml` we have only put the `8.1` version in the `include` section to run it with code coverage (we cannot run unit tests against WP `trunk`). This is why there are differences between the lines.

We can confirm that things work correctly by going to the PR's page and verifying that integration and unit tests run for PHP 7.4 - 8.3, which they do.

Reply:
Our plugin supports PHP version 7.2 and up, so we still need to test against 7.2.

But we'll remove versions as our PHP requirements go up.